wordpress-mobile/WordPressMocks
# Mocking changes for WordPress iOS UI and screenshot tests

This PR makes the necessary changes to the mocks definitions for compatibility with WordPress-iOS UI and screenshot tests (see https://github.com/wordpress-mobile/WordPress-iOS/pull/11788).

WPiOS and WPAndroid differ in their network activity in several subtle ways. In some cases one uses a trailing slash where the other doesn't while in others they use different API endpoints or versions. This allows the mocks to work with both.

### To Test

- Run `./scripts/start.sh` and ensure there are no errors.
- Follow the testing steps in https://github.com/wordpress-mobile/WordPress-iOS/pull/11788 which uses this branch.
